__tests__/app.test.js

```diff
@@ -192,6 +192,70 @@ describe('GET /api/articles/:article_id/comments', () => {
   });
 });
 
+describe('POST /api/articles/:article_id/comments', () => {
+  it('201: should post a new comment and then return a comment object upon being added successfully', () => {
+    return request(app)
+      .post('/api/articles/1/comments')
+      .send({ username: 'rogersop', body: 'test comment' })
+      .expect(201)
+      .then(({ body }) => {
+        const { comment } = body;
+
+        expect(comment).toMatchObject({
+          article_id: 1,
+          author: 'rogersop',
+          body: 'test comment',
+          votes: 0,
+          comment_id: expect.any(Number),
+          created_at: expect.any(String),
+        });
+      });
+  });
+  it('422: should return a unprocessable content error when an non-existant article id is provided', () => {
+    return request(app)
+      .post('/api/articles/999/comments')
+      .send({ username: 'rogersop', body: 'test comment' })
+      .expect(422)
+      .then(({ body }) => {
+        const { error } = body;
+
+        expect(error).toHaveProperty('msg', 'Unprocessable Content');
+      });
+  });
+  it('422: should return a unprocessable content error when an non-existant username is provided', () => {
```

Comment:
A 422 is when a request is well-formed, however, due to semantic errors it is unable to be processed, I'm not 100% sure this is a semantic error and a 404 would also work fine here, but it's up to you!